cmd/main_branch.go

```diff
@@ -0,0 +1,34 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/Originate/git-town/lib/git"
+	"github.com/spf13/cobra"
+)
+
+var mainBranchCommand = &cobra.Command{
+	Use:   "main-branch [<branch>]",
+	Short: "Displays or sets your main branch",
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) == 0 {
+			output := git.GetMainBranch()
```

Comment:
Extract the "true" path of the if clause into an `printMainBranch` method, and the "false" path into an `setMainBranch` method.